controllers/admin/AdminShoprunbackController.php

```diff
@@ -270,10 +274,10 @@ private function getConfigFormValues()
         $this->context->smarty->assign('production', Configuration::get('production'));
     }
 
-    public function asyncCall()
+/*     public function asyncCall()
     {
         require_once($this->module->SRBModulePath . '/asyncCall.php');
-    }
+    } */
```

Comment:
if not needed anymore and no regression, you can remove the commented code in the next PR